Compilation fails: missing string

I use the following cmake line:

cmake -DUSB_SERIAL=OFF -DGVARS=NO -DLUA_COMPILER=NO -DMULTIMODULE=NO -DDEBUG=NO -DHELI=OFF -DTRANSLATIONS=DE -DPCB=I6X -DOVERRIDE_CHANNEL_FUNCTION=OF
F -DDISABLE_COMPANION=YES ..

but I get the following error:

/home/lmeier/Projekte/openi6x/opentx/radio/src/telemetry/crossfire.cpp:42:21: error: 'ZSTR_BATT_PERCENT' was not declared in this scope; did you mean 'ZSTR_BATT1_CURRENT'?
42 | {BATTERY_ID, 3, ZSTR_BATT_PERCENT, UNIT_PERCENT, 0},
| ^~~~~~~~~~~~~~~~~
| ZSTR_BATT1_CURRENT
make[2]: *** [radio/src/CMakeFiles/firmware.dir/build.make:370: radio/src/CMakeFiles/firmware.dir/telemetry/crossfire.cpp.o] Fehler 1
make[1]: *** [CMakeFiles/Makefile2:493: radio/src/CMakeFiles/firmware.dir/all] Fehler 2

--- a/radio/src/translations/de.h.txt
+++ b/radio/src/translations/de.h.txt
@@ -98,8 +98,17 @@
 #define LEN_COUNTRYCODES       TR("\002", "\007")
 #define TR_COUNTRYCODES        TR("US""JP""EU", "Amerika""Japan\0 ""Europa\0")
 
+#if defined(PCBI6X)
+#define LEN_USBDETECTMODES             TR("\004", "\004")
+#define TR_USBDETECTMODES              TR("Auto""Once", "Auto""Once")
+#endif
+
 #define LEN_USBMODES           TR("\006", "\010")
+#if defined(PCBI6X)
+#define TR_USBMODES            TR("Fragen""Joyst\0""Storag", "Fragen\0 ""Joystick""Speicher")
+#else
 #define TR_USBMODES            TR("Fragen""Joyst\0""Storag""Serial", "Fragen\0 ""Joystick""Speicher""Seriell\0")
+#endif
 
 #define LEN_TARANIS_PROTOCOLS  "\004"
 #if defined(PCBI6X)
@@ -232,7 +241,7 @@
   #define TR_VVARIO            "[Vario]\0  "
 #endif
 
-#if defined(AUDIO)
+#if defined(AUDIO) || defined(BUZZER)
   #define TR_SOUND             "Spiel Töne"
 #else
   #define TR_SOUND             "Spiel\0    "
@@ -436,7 +445,10 @@
 #elif defined(PCBX9E)
   #define TR_POTS_VSRCRAW              "\310F1\0""\310F2\0""\310F3\0""\310F4\0""\311S1\0""\311S2\0""\311LS\0""\311RS\0"
   #define TR_SW_VSRCRAW                "\312SA\0""\312SB\0""\312SC\0""\312SD\0""\312SE\0""\312SF\0""\312SG\0""\312SH\0""\312SI\0""\312SJ\0""\312SK\0""\312SL\0""\312SM\0""\312SN\0""\312SO\0""\312SP\0""\312SQ\0""\312SR\0"
-#elif defined(PCBXLITE) || defined(PCBI6X)
+#elif defined(PCBI6X)
+  #define TR_POTS_VSRCRAW              "\310P1\0""\310P2\0"
+  #define TR_SW_VSRCRAW                "\312SA\0""\312SB\0""\312SC\0""\312SD\0"
+#elif defined(PCBXLITE)
   #define TR_POTS_VSRCRAW              "\310S1\0""\310S2\0"
   #define TR_SW_VSRCRAW                "\312SA\0""\312SB\0""\312SC\0""\312SD\0"
 #elif defined(PCBX7)
@@ -541,12 +553,20 @@
 #endif
 
 #if defined(PCBTARANIS) || defined(PCBHORUS)
-  #define TR_ENTER             "[ENTER]"
+  #define TR_ENTER                     "[ENTER]"
+#elif defined(PCBI6X)
+  #define TR_ENTER                     "[ OK ]"
 #else
-  #define TR_ENTER             "[MENU]"
+  #define TR_ENTER                     "[MENU]"
 #endif
 
-#define TR_EXIT                "[EXIT]"
+#if defined(PCBHORUS)
+  #define TR_EXIT                      "[RTN]"
+#elif defined(PCBI6X)
+  #define TR_EXIT                      "[CANCEL]"
+#else
+  #define TR_EXIT                      "[EXIT]"
+#endif
 
 #if defined(PCBTARANIS)
   #define TR_POPUPS            TR_EXIT "\010" "\010" "\010" "\010" TR_ENTER
@@ -937,6 +957,7 @@
 #define TR_MENUSENSOR          "SENSOR"
 #define TR_SENSOR              "SENSOR"
 #define TR_COUNTRYCODE         "Landescode"
+#define TR_USBDETECT           "USB Detect"
 #define TR_USBMODE             "USB Modus"
 #define TR_VOICELANG           "Sprach-Ansagen"
 #define TR_UNITSSYSTEM         "Einheiten"
@@ -1217,6 +1238,7 @@
 #define ZSTR_CURR              "Curr"
 #define ZSTR_CAPACITY          "Capa"
 #define ZSTR_VFAS              "VFAS"
+#define ZSTR_BATT_PERCENT      "Bat%"
 #define ZSTR_ASPD              "ASpd"
 #define ZSTR_GSPD              "GSpd"
 #define ZSTR_HDG               "Hdg"

--- a/radio/src/translations/en.h.txt
+++ b/radio/src/translations/en.h.txt
@@ -104,7 +104,11 @@
 #endif
 
 #define LEN_USBMODES                   TR("\006", "\010")
+#if defined(PCBI6X)                    
+#define TR_USBMODES                    TR("Ask\0  ""Joyst\0""Storag", "Ask\0    ""Joystick""Storage\0")
+#else
 #define TR_USBMODES                    TR("Ask\0  ""Joyst\0""Storag""Serial", "Ask\0    ""Joystick""Storage\0""Serial\0 ")
+#endif
 
 #define LEN_TARANIS_PROTOCOLS  "\004"
 #if defined(PCBI6X)
@@ -180,7 +184,6 @@
 #if defined(IRPROTOS)
   #define TR_IRPROTOS                  "SILV  TRAC09PICZ  SWIFT\0"
 #else
-
   #define TR_IRPROTOS
 #endif
 
@@ -238,11 +241,11 @@
   #define TR_VVARIO                    "[Vario]\0  "
 #endif
 
-//#if defined(AUDIO)
+#if defined(AUDIO) || defined(BUZZER)
   #define TR_SOUND                     "Play Sound"
-//#else
-//  #define TR_SOUND                     "Beep\0     "
-//#endif
+#else
+  #define TR_SOUND                     "Beep\0     "
+#endif
 
 #if defined(HAPTIC)
   #define TR_HAPTIC                    "Haptic\0   "
@@ -1088,11 +1091,6 @@
 // About screen
 #define TR_ABOUTUS                     TR(" ABOUT ", "ABOUT")
 
-#if defined(PCBI6X)
-#define TR_ABOUT_OPENTX_1              TR("OpenI6X", "")
-#define TR_ABOUT_OPENTX_2              TR("is\001based\001on\001OpenTX", "")
-
-#else
 #define TR_ABOUT_OPENTX_1              TR("OpenTX\001is\001open\001source,\001non", "OpenTX is open source, non-")
 #define TR_ABOUT_OPENTX_2              TR("commercial,\001no\001warranties.", "commercial and comes with no")
 #define TR_ABOUT_OPENTX_3              TR("It\001was\001developed\001for\001free.", "warranties. It was developed")
@@ -1141,8 +1139,6 @@
 #define TR_ABOUT_PARENTS_2             TR("Ersky9x (Mike Blandford)", "Ersky9x (Mike Blandford)")
 #define TR_ABOUT_PARENTS_3             "ER9X (Erez Raviv)"
 #define TR_ABOUT_PARENTS_4             "TH9X (Thomas Husterer)"
-
-#endif
 
 #define TR_CHR_SHORT                   's'
 #define TR_CHR_LONG                    'l'

